src/virtual-source-buffer.js

```diff
@@ -154,6 +154,12 @@ export default class VirtualSourceBuffer extends videojs.EventTarget {
         return videojs.createTimeRanges(ranges);
       }
     });
+
+    Object.defineProperty(this, 'videoTracks', {
```

Comment:
We can do that, but we'd have to point a new local property called videoTracks at videoBuffer_.videoTrack.